packages/react-inbox/src/components/Inbox/LazyTippy.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { forwardRef, useState } from "react";
+import Tippy, { TippyProps } from "@tippyjs/react";
+
+const LazyTippy: React.FunctionComponent<TippyProps> = forwardRef(
+  (props, ref) => {
+    const [mounted, setMounted] = useState(false);
+
+    const lazyPlugin = {
+      fn: () => ({
+        onMount: () => setMounted(true),
+        onHidden: () => setMounted(false),
+      }),
+    };
+
+    const computedProps = { ...props };
+
+    computedProps.plugins = [lazyPlugin, ...(props.plugins || [])];
```

Comment:
`lazyPlugin` is an object, so computedProps.plugins is an array of objects.
But I am confused on what `...(props.plugins || [])` resolves to.
 Maybe some typedefs here